src/main.rs

```diff
@@ -43,7 +48,80 @@ const LINE_END: &str = "\n";
 const LINE_END: &str = "\r\n";
 
 fn main() -> ExitCode {
-    let mut args = Cli::parse();
+    // parse CLI arguments
+    let cli_args = Cli::parse();
+
+    // initilize default config
+    let default_options = Cli {
+        check: false,
+        print: false,
+        keep: false,
+        verbose: false,
+        stdin: false,
+        quiet: false,
+        trace: false,
+        files: Vec::<String>::new(),
+        tab: 2,
+        usetabs: false,
+        wrap: 80,
+        wrap_min: 70,
+    };
```

Comment:
Just a small suggestion: this could be condensed by implementing (or maybe just deriving) the `Default` trait for `Cli` and calling `default()`.
```suggestion
    // initilize default config
    let default_options = Cli::default();
```